x-pack/plugins/secops/public/components/timeline/body/renderers/formatted_field.tsx

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import moment from 'moment';
+import * as React from 'react';
+import { pure } from 'recompose';
+
+import { Ecs } from '../../../../../server/graphql/types';
+import { getMappedEcsValue } from '../../../../lib/ecs';
+import { getOrEmptyTag } from '../../../empty_value';
+import { LocalizedDateTooltip } from '../../../localized_date_tooltip';
+
+export const FormattedField = pure<{ data: Ecs; fieldName: string; fieldType: string }>(
+  ({ data, fieldName, fieldType }) => {
+    const value = getMappedEcsValue({ data, fieldName });
+    const maybeDate = moment(new Date(value!));
+
+    return (
+      <>
+        {fieldType === 'date' && maybeDate.isValid() ? (
+          <LocalizedDateTooltip date={maybeDate.toDate()}>{value}</LocalizedDateTooltip>
+        ) : (
+          getOrEmptyTag(fieldName, data)
```

Comment:
Just letting you know this will silently fail if date time is a Epoch which comes back from GraphQL as a string:

```ts
moment(new Date("1550681832588")).isValid()
false
```
Which we have seen before and written tickets for it.

Likewise `null` comes back as an ok date time

```ts
moment(new Date(null)).isValid()
true
```

And you'll get a 1970 Epoch value.